benchmarks/prescribed_plastic_dilation/brick_extension_prescribed_plastic_dilation_explicit.prm

```diff
@@ -0,0 +1,214 @@
+#  Global parameters
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 0e3
+set Use years in output instead of seconds = true
+#set Nonlinear solver scheme                = single Advection, iterated Newton Stokes
+set Nonlinear solver scheme                = single Advection, iterated Stokes
```

Comment:
did you try defect correction?

Reply:
I tried Newton, and the convergence was not great, but will try again.

Reply:
Overall, the problem is very difficult to solve with prescribed dilation on. It would be good to compare convergence rates with @cedrict.